ospfd/ospf_vty.c

```diff
@@ -1548,21 +1537,60 @@ DEFUN (ospf_area_nssa_no_summary,
        "Configure OSPF area as nssa\n"
        "Do not inject inter-area routes into nssa\n")
 {
-	return ospf_area_nssa_cmd_handler(vty, argc, argv, 1);
+	int idx_ipv4_number = 1;
+	struct in_addr area_id;
+	int format;
+
+	VTY_DECLVAR_INSTANCE_CONTEXT(ospf, ospf);
+	VTY_GET_OSPF_AREA_ID_NO_BB("NSSA", area_id, format,
+				   argv[idx_ipv4_number]->arg);
+
+	ospf_area_display_format_set(ospf, ospf_area_get(ospf, area_id),
+				     format);
+	ospf_area_nssa_no_summary_set(ospf, area_id);
+
```

Comment:
Missing call to `ospf_schedule_abr_task` here like for `no_opsf_area_nssa_no_summary` command. Please add this line:

`ospf_schedule_abr_atsk(ospf);`